src/main/java/io/cryostat/net/NetworkModule.java

```diff
@@ -152,7 +152,7 @@ static Vertx provideVertx() {
         return Vertx.vertx(
                 new VertxOptions()
                         .setPreferNativeTransport(true)
-                        .setEventLoopPoolSize(defaults.getEventLoopPoolSize() + 4));
+                        .setEventLoopPoolSize(defaults.getEventLoopPoolSize() + 6));
```

Comment:
What is the choice of the additional 4 threads previously and 6 here?

Reply:
It's pretty hacky.

https://github.com/cryostatio/cryostat/pull/1658

I noticed that the Podman discovery was broken again with this PR even after rebasing on top of that one. I haven't chased exactly why or what is causing the additional blocking work at startup, but there is some thread blockage going on that prevents the platform discovery strategy check from correctly finishing, and increasing the size of the worker pool allows the check to succeed.

Reply:
hmm yehhh make sense! Probably, we should document it somewhere.

Tho, in environment where resources are more relaxed, I suppose these settings might just be a little "overkill"?

Reply:
Vertx's default setting is to have `cpuCores * 2` threads in the pool, so definitely adding a bunch of extra threads here like this is going to increase the size of the pool substantially for most deployments. The proper fix would be to avoid doing blocking work on startup, or at least move that blocking work off of a Vertx event loop thread, but that's a relatively deep change and I haven't had enough time to look into it yet. If I have time before the next 2.4 release I will, but it seems riskier to do that than to simply bump up the minimum pool size a bit for now.

Reply:
Sounds good! I think we can file a separate issue to track. Perhaps, this could be fixed in 3.0.

Reply:
It will already be fundamentally different in 3.0 since the whole startup mechanism is completely different. I think the 3.0 prototype already completely avoids this kind of problem...

Reply:
Oh yes right that just slipped me mind. Thanks :))